### PR TITLE
Bring up vlan interfaces on creation

### PR DESCRIPTION
**Pull request by freqlabs**
Issue: NAS-100773

### CODE REVIEW

**Comment by freqlabs** [comment]
When an interface is created in the UI, middleware does bring the interface up when creating it. However, middleware doesn't create the interfaces during system startup, netif does.

**Comment by william-gr** [comment]
You didnt understand my question. Middleware should create the interface with "up" in int_options database field, which would put it in rc.conf already.

Let me rephrase. Why do you need to `up` the interface at creation time, and not on configure time?

**Comment by freqlabs** [comment]
Oh got it, yes "up" is not in the interface options in the config database, you're right.

**Comment by freqlabs** [comment]
This should be fixed in middleware instead.
